DESCRIPTION

```diff
@@ -71,7 +71,7 @@ Suggests:
     mapproj,
     rappdirs,
```

Comment:
Please add yourself as an author.

Reply:
Dean is listed as an author already in the version on CRAN now and in the devel version here on GitHub where I've been merging his PR.

Reply:
Blergh. Sorry -- I thought because the PR came from an external repo.